src/test/java/com/conveyal/datatools/manager/models/transform/NormalizeFieldTransformationTest.java

```diff
@@ -106,6 +106,24 @@ private static Stream<Arguments> createSubstitutionCasesWithOwnExceptions() {
         );
     }
 
+
+    @ParameterizedTest
+    @MethodSource("createCreateNewField")
+    void testCreateNewField(String input, String expected) {
+        NormalizeFieldTransformation transform = createTransformation(
+                "table", "new_field",  null, Lists.newArrayList(
+                        new Substitution("", "new_value")
+                ));
+        assertEquals(expected, transform.performSubstitutions(input));
+    }
+
+    private static Stream<Arguments> createCreateNewField() {
```

Comment:
Rename to `createNewField`.

Reply:
But this method creates a method that creates a new field! Doesn't it? Maybe I confused myself